core/templates/core/login.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'core/base.html' %}
+
+{% block title %}Logowanie{% endblock %}
+
+{% block content %}
+<script type="module">
+    const canvas = document.getElementById('starback-bg');
+    const starback = new Starback(canvas, {
+        type: 'dot',
+        quantity: 250,
+        height: window.innerHeight,
+        width: window.innerWidth,
+        backgroundColor: [],
+        randomOpacity: true,
+    });
+    function resizeCanvas() {
+        canvas.width = window.innerWidth;
+        canvas.height = window.innerHeight;
+    }
+
+    window.addEventListener('resize', resizeCanvas);
+</script>
+<canvas id="starback-bg" class="fixed -z-1" width="871" height="911"></canvas>
+<div class="flex justify-center grid justify-items-center bg-transparent place-content-center">
+    <div class="h-screen w-screen relative flex flex-col items-center justify-center text-center">
```

Comment:
Why position relative and flex simultanously?? Also what is the reason for creating this div? You can move all classes to its parent and delete it, it doesn't change anything

Reply:
I don't want to destroy something. "Why position relative and flex simultanously??" - IDK